libavformat/segment.c

```diff
@@ -984,6 +1000,10 @@ static int seg_write_packet(AVFormatContext *s, AVPacket *pkt)
     /* compute new timestamps */
     offset = av_rescale_q(seg->initial_offset - (seg->reset_timestamps ? seg->cur_entry.start_pts : 0),
                           AV_TIME_BASE_Q, st->time_base);
+
+    av_log(s, AV_LOG_DEBUG, "offset=%"PRId64", reset_timestamps=%d, cur_entry.start_pts=%"PRId64", initial_offset=%"PRId64", pkt->pts=%"PRId64,
```

Comment:
This log will be too much for streaming - an you ifdef it out before commit and we can use it in custom builds?

Reply:
it is debug and it will not impact prod build in qfab (otherwise you turn debug on avpipe and do some changes in there).